community/modules/scheduler/schedmd-slurm-gcp-v5-controller/outputs.tf

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "controller_instance_id" {
+  description = "Self link of the controller node"
+  value       = module.slurm_controller_instance.slurm_controller_instance.instances_details[0].id
```

Comment:
I do not believe that this is, in fact, the self_link of the instance.

https://github.com/SchedMD/slurm-gcp/blob/master/terraform/slurm_cluster/modules/_slurm_instance/outputs.tf#L23

Reply:
I think you might consider output-ing the ID and self-link.

Reply:
It looks like I may not need the controller module to be output after all. It was needed before to specify the dependency, but it looks like that was fixed in recent updates.
I'll run some tests and if it works without adding the explicit `depends_on`, I'll convert this to self_link.

Reply:
The module output is no longer needed, I removed it. In the process I also fixed a bug related to filestore, so I'll be running a larger set of tests to ensure that didn't break anything else.